doc/doc_utils/jumpstart_doc_utils.py

```diff
@@ -159,8 +162,15 @@ def get_jumpstart_sdk_manifest():
     return json.loads(models_manifest)
 
 
+def get_proprietary_sdk_manifest():
+    url = "{}/{}".format(PROPRIETARY_DOC_BUCKET, PROPRIETARY_SDK_MANIFEST_FILE)
+    with request.urlopen(url) as f:
+        models_manifest = f.read().decode("utf-8")
+    return json.loads(models_manifest)
+
+
 def get_jumpstart_sdk_spec(key):
-    url = "{}/{}".format(JUMPSTART_BUCKET_BASE_URL, key)
+    url = "{}/{}".format(PROPRIETARY_DOC_BUCKET, key)
```

Comment:
sanity-check: is this change correct?

Reply:
Thanks for catching, this is not intended but should be no harm, updated.